src/explore-education-statistics-common/src/modules/table-tool/utils/createExpandedRowHeaders.ts

```diff
@@ -91,31 +91,33 @@ function getRowHeadersInfo(rowHeaders: Header[]): {
 
   rowHeaders.forEach(header => {
     const { maxCrossSpan } = header;
+    const stack = [header];
```

Comment:
Nice use of DFS. I did wonder if a BFS might be more beneficial somehow? Haven't look too hard into it, but maybe something to consider as well.